.stylish-haskell.yaml

```diff
@@ -15,41 +15,13 @@ steps:
   #     # true.
   #     add_language_pragma: true
 
-  # Format record definitions. This is disabled by default.
```

Comment:
This should probably be squashed down into one of the previous commits.

Reply:
Squashed!